sdk/storage/azure-storage-blobs/swagger/README.md

```diff
@@ -694,6 +682,21 @@ directive:
 
 ```yaml
 directive:
+  - from: swagger-document
+    where: $.definitions
+    transform: >
+      $.AccountInfo = {
```

Comment:
You didn't delete Line 151
```
"Service_GetAccountInfo": "AccountInfo",
```

Does this patch under `$.definitions` have any effect?

Reply:
Yeah, we need to change the description of `AccountInfo ` to match doxygen. (`ServiceClient` -> `BlobServiceClient`)

Reply:
Removed.